src/QueryBuilder/DSL/Aggregation.php

```diff
@@ -422,13 +422,12 @@ public function adjacency_matrix(string $name): AdjacencyMatrix
         return new AdjacencyMatrix($name);
     }
 
-    /** sampler aggregation.
+    /**
+     * sampler aggregation.
      *
      * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-sampler-aggregation.html
-     *
-     * @param string $name
      */
-    public function sampler($name): Sampler
+    public function sampler(string $name): Sampler
```

Comment:
BC break as type-hint is changed, but this is currently breaking when calling the constructor with another type-hint as constructor only accepts a `string` for name argument.